build.rs

```diff
@@ -136,7 +136,7 @@ use std::fmt::Write;
         s.extend(87..89);
         s.extend(91..93);
         s.extend(94..105);
-        s.extend(106..113);
+        s.extend(105..113);
```

Comment:
This can be consolidated with the line above it.

Reply:
Dooh, thanks for catching that.